.vitepress/config.mts

```diff
@@ -40,7 +40,7 @@ export default defineConfig({
               link: '/overview/tokenomics.md',
             },
             {
-              text: 'Web UI',
+              text: 'Web App UI',
```

Comment:
please check all links once again, I can see `Tutorials: API -> Retrieving top contributions -> 404`, moreover let's get rid of `top`